packages/rspack/src/config/builtins.ts

```diff
@@ -57,7 +63,7 @@ export interface Builtins {
 	provide?: Record<string, string | string[]>;
 	html?: Array<BuiltinsHtmlPluginConfig>;
 	decorator?: boolean | Partial<RawDecoratorOptions>;
-	minifyOptions?: Partial<RawMinification>;
+	minifyOptions?: MinificationConfig;
```

Comment:
Does this mean we can't use the swc minify options and `extractComments` at the same time?

Reply:
https://github.com/web-infra-dev/rspack/pull/2882/files#diff-fd42ec611cd5acb13724f10d47d47067d31c35ae5a06062825c02fb81a1d8450R140 
I think I know why you define the same type twice.

Reply:
I just want replace the `extractComments` option, but seems it doesn't work as I expected